packages/react-core/src/components/Form/FormGroupLabelHelp.tsx

```diff
@@ -10,15 +11,47 @@ export interface FormGroupLabelHelpProps extends ButtonProps {
   'aria-label': string;
   /** Additional classes added to the help button. */
   className?: string;
+  /** @hide Forwarded ref */
+  innerRef?: React.Ref<HTMLSpanElement>;
 }
 
-export const FormGroupLabelHelp: React.FunctionComponent<FormGroupLabelHelpProps> = ({
+const FormGroupLabelHelpBase: React.FunctionComponent<FormGroupLabelHelpProps> = ({
   'aria-label': ariaLabel,
   className,
+  innerRef,
   ...props
-}) => (
-  <Button aria-label={ariaLabel} className={className} variant="plain" hasNoPadding {...props}>
-    <QuestionCircleIcon />
-  </Button>
-);
+}) => {
+  const ref = React.useRef<HTMLSpanElement>(null);
+  const buttonRef = innerRef || ref;
+
+  const isMutableRef = (ref: React.Ref<HTMLSpanElement>): ref is React.MutableRefObject<HTMLSpanElement> =>
+    typeof ref === 'object' && ref !== null && 'current' in ref && ref.current !== undefined;
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLSpanElement>) => {
+    if ([KeyTypes.Space, KeyTypes.Enter].includes(event.key) && isMutableRef(buttonRef) && buttonRef.current) {
+      buttonRef.current.click();
+    }
+  };
```

Comment:
One last little thing. When pressing Space to trigger the popover in this instance, we'll need to prevent the page from scrolling